bentoml/frameworks/spark_mllib.py

```diff
@@ -0,0 +1,128 @@
+import importlib
+import json
+import logging
+import os
+
+from bentoml.exceptions import BentoMLException, MissingDependencyException
+from bentoml.service import BentoServiceArtifact, BentoServiceEnv
+
+
+class PySparkModelArtifact(BentoServiceArtifact):
+    """
+    PySparkModelArtifact allows you to use the spark.ml and spark.mllib APIs with Bento
+
+    Example usage:
+
+    >>> from pyspark.sql import SparkSession
+    >>>
+    >>> spark = SparkSession.builder.getOrCreate()
+    >>>
+    >>> # Load training data
+    >>> training = spark.read.format("libsvm").load("data/mllib/sample_libsvm_data.txt")
+
```

Comment:
have you tried viewing this docstring with `make watch-doc` locally? this empty line here could break the formatting, you might want to add the leading `>>>`  even for empty lines in the code block in docstring